components/batch-upload/SnapshotUpload.tsx

```diff
@@ -289,7 +300,13 @@ const SnapshotTable: React.FC<SnapshotTableProps> = ({ data, portalRef }) => {
           }
         },
       },
-    ] as Column<{ address: string }>[];
+      {
+        Header: "Max claimable per transaction",
+        accessor: ({ maxClaimable }) => {
+          return maxClaimable || "Default";
```

Comment:
probably just ` || 1` here, no? (that *is* the default?)

Reply:
No it's not, it defaults to whatever you have set on max claimable in the claim phase
![image](https://user-images.githubusercontent.com/50103937/164800311-fd8d2fe3-38fe-4572-ab04-798585e088eb.png)

Reply:
ah, fair enough then